config/config.go

```diff
@@ -21,6 +22,9 @@ type Config struct {
 	FeedbackEnabled                     bool          `envconfig:"FEEDBACK_ENABLED"`
 	SearchControllerURL                 string        `envconfig:"SEARCH_CONTROLLER_URL"`
 	SearchRoutesEnabled                 bool          `envconfig:"SEARCH_ROUTES_ENABLED"`
+	ReleaseCalendarControllerURL        string        `envconfig:"RELCAL_CONTROLLER_URL"`
+	ReleaseCalendarEnabled              bool          `envconfig:"RELCAL_ENABLED"`
+	ReleaseCalendarPrivateRoutePrefix   string        `envconfig:"RELCAL_PRIVATE_ROUTE_PREFIX"`
```

Comment:
May I suggest we use `RELEASE_CALENDAR` prefix instead of `RELCAL` so to be in line with what we do [in the api router](https://github.com/ONSdigital/dp-api-router/blob/2a210dbf51af0b0ac0e9006d79482dff543473d9/config/config.go#L62-L63)?

Reply:
yeah can do